src/sysdolphin/baselib/forward.h

```diff
@@ -0,0 +1,6 @@
+#ifndef SYSDOLPHIN_BASELIB_FORWARD_H
+#define SYSDOLPHIN_BASELIB_FORWARD_H
+
+typedef struct _HSD_Obj HSD_Obj;
+
+#endif
```

Comment:
Just a quick fix to eliminate `object.h`⮀`class.h` circular dependency. Will do a full forward declaration sweep later.